src/sg/gov/moh/healthcert/1.0/interim-healthcert-schema.json

```diff
@@ -0,0 +1,703 @@
+          },
+          "name": {
+            "type": "string",
+            "examples": ["SAMPLE CLINIC"]
+          },
+          "identityProof": {
+            "type": "object",
+            "required": ["type", "location", "key"],
+            "properties": {
+              "type": { "type": "string" },
+              "location": {
+                "type": "string",
+                "examples": ["donotverify.testing.verify.gov.sg"]
+              },
+              "key": {
+                "type": "string",
+                "examples": ["did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830#controller"]
+              }
+            },
+            "examples": [
+              {
+                "type": "DNS-DID",
+                "location": "donotverify.testing.verify.gov.sg",
+                "key": "did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830#controller"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "logo": {
+      "type": "string",
+      "description": "base64 encoded image"
+    },
+    "$template": {
+      "type": "object",
+      "required": ["name", "type"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Template name to be use by template renderer to determine the template to use",
+          "examples": ["HEALTHCERT"]
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of renderer template",
+          "enum": ["EMBEDDED_RENDERER"]
+        },
+        "url": {
+          "type": "string",
+          "description": "URL of a decentralised renderer to render this document",
+          "examples": ["renderer.example.com"]
+        }
+      }
+    },
+    "notarisationMetadata": {
+      "type": "object",
+      "properties": {
+        "reference": { "type": "string", "examples": ["e45acb48-a782-4883-b4df-c06129f6e783"] },
+        "notarisedOn": { "type": "string", "format": "date-time", "examples": ["2021-05-17T09:01:49.836Z"] },
+        "passportNumber": { "type": "string", "examples": ["E7831177G"] },
+        "url": {
+          "type": "string",
+          "examples": [
+            "https://www.verify.gov.sg/verify?q=%7B%22type%22%3A%22DOCUMENT%22%2C%22payload%22%3A%7B%22uri%22%3A%22https%3A%2F%2Fapi.storage.staging.notarise.io%2Fdocument%2F2d0cff63-8b0f-4056-a4b4-ef585057bd00%22%2C%22key%22%3A%2221a6e3251229bf7d0c4e4b8cc603d2026c310c24fbcd9e94cb24f87e25897c06%22%2C%22permittedActions%22%3A%5B%22VIEW%22%2C%22STORE%22%5D%7D%7D"
+          ]
+        }
+      },
+      "additionalProperties": true
+    },
+    "attachments": {
+      "type": "array",
+      "items": {
+        "properties": {
+          "filename": { "type": "string", "examples": ["healthcert.txt"] },
+          "type": { "type": "string", "enum": ["text/open-attestation"] },
+          "data": { "type": "string" }
+        },
+        "additionalProperties": true
+      },
+      "additionalItems": true
```

Comment:
remove this line when single schema for array. Same for all other `additionalItems` in this schema.
see https://json-schema.org/understanding-json-schema/reference/array.html